jax/experimental/jax2tf/README.md

```diff
@@ -194,19 +197,20 @@ For examples of how to save a Flax model as a SavedModel see the
 
 ### Saved model and differentiation
 
-The converted code supports differentiation from TensorFlow. In order to
+The code lowered from JAX supports differentiation from TensorFlow. In order to
 ensure that the result of TensorFlow differentiation is identical to the
-one that JAX differentiation would produce, the jax2tf converter will
-annotate the converter function with a ``tf.custom_gradient`` that,
+one that JAX differentiation would produce, the jax2tf decorator will
```

Comment:
I am not sure if "decorator" is accurate here (and in other places), since jax2tf can be used as a decorator but also by using the function directly. Also there seems to be no decoration happening in the examples in the previous section.

Reply:
I have removed the references to `decorator`.